### ISSUE TITLE
Feature request: Doors, Trunk, Window Open/Close Status

### ISSUE THREAD

**Issue by KumarMudedla** [samrum/OnStarJS#262]
Thank you for all this great work, trying to see if there is any way we can get Doors, Trunk, Window Open/Close Status ? It's feature request not an issue

**Comment by joelvandal** [comment]
At my knowledge, and during all my testing and capture, it not kind of infos that OnStar send about current status (ex. door is unlocked, window is open, etc)

We can lock/unlock door/trunk but not window.

**Comment by KumarMudedla** [comment]
Correct, it's a limitation on the OnStar, but trying to see if any other possibilities to get door lock status, to quickly review on the dashboard  (home assistant)
